docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -149,7 +149,7 @@ class [[eosio::contract]] smrtcontract : public contract {
 
 The property used for the second parameter must be of template type `std::tuple`. The first parameter must be the type of the property indexed non-uniquely, in our case the type `std::string` is used because `first_name` is the property indexed non-uniquely. And the last parameter of the tuple type must be the type of a property name which is unique. In our case the type `eosio::name` is used because property `account_name` is unique. Multiple properties can be indexed non-uniquely as well. In this case the first parameters types correspond to the properties being indexed. And, as previously already mentioned, the last parameter correspond to the type of a property name which is unique.
```

Comment:
parameter type not parameters types

Reply:
+ And, as previously already mentioned, the last parameter correspond to the type of a property name which is unique.

could be

The last parameter corresponds to the type of a property name. The property name is unique. - or something like this

Reply:
Both the paragraphs describing adding non-unique indexes are not clear/easy to understand

Reply:
also i can't see what the difference between declaring unique and non-unique indexes ... is this implicit/depend on the  data being indexed?

Reply:
non-unique vs unique indexes for kv tables are now pretty convoluted
they will be revamped in future version